### PR TITLE
double rat kebabs are now double the reagents of single rat kebabs also you eat the stick

### PR DESCRIPTION
**Pull request by Kylerace**
## About The Pull Request
FIX

## Changelog
:cl:
fix: double rat kebabs are now double the reagents of single rat kebabs
fix: rat kebabs no longer drop the metal rods when eaten unlike other kebabs to maintain lore consistency 
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
image
This violates my design vision

**Comment by LemonInTheDark** [comment]
The pact is sealed

**Review by Twaticus — APPROVED** [review]
eat rod

**Review by LemonInTheDark — APPROVED** [review]

**Comment by RaveRadbury** [comment]
> eat rod

image

**Comment by Kylerace** [comment]
> 
> 
> > eat rod
> 
> image

this is even more evidence that this is the intended functionality because i didnt even add that

**Comment by RaveRadbury** [comment]
> > > eat rod
> > 
> > 
> > image
> 
> this is even more evidence that this is the intended functionality because i didnt even add that

Add iron reagent.

File: code/game/objects/items/food/meat.dm [hunk 1]
```diff
@@ -575,7 +576,7 @@
 	name = "double rat-kebab"
 	icon_state = "doubleratkebab"
 	tastes = list("rat meat" = 2, "metal" = 1)
-	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 16, /datum/reagent/consumable/nutriment/vitamin = 6)
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 20, /datum/reagent/consumable/nutriment/vitamin = 4)
```
Comment (interestingusernam3) [inline code/game/objects/items/food/meat.dm:579]: That's not double.

Comment (Kylerace) [reply code/game/objects/items/food/meat.dm:579]: read the title again

Comment (Qustinnus) [inline code/game/objects/items/food/meat.dm:579]: this is not how food code works, now its way more than double.

Comment (Kylerace) [reply code/game/objects/items/food/meat.dm:579]: i was told from up high to make the numbers 2x as big so i did that

Comment (Mothblocks) [reply code/game/objects/items/food/meat.dm:579]: @Qustinnus can you elaborate because i dont know what you mean

Comment (Qustinnus) [reply code/game/objects/items/food/meat.dm:579]: nvm im wrong, i thought this was the food thats crafted from the regular rat kebab, which would've made it add reagents ontop of the original reagents.

**Comment by Zytolg** [comment]
Iron is healthy for the diet

File: code/game/objects/items/food/meat.dm (hunk 1 above)
Comment (maxymax13) [inline code/game/objects/items/food/meat.dm:579]:
```suggestion
	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 20, /datum/reagent/consumable/nutriment/vitamin = 4, /datum/reagent/iron = 2)
```

**Pull request merged**